src/main/scala/org/neo4j/spark/Executor.scala

```diff
@@ -1,32 +1,19 @@
 package org.neo4j.spark
 
-import java.sql.Timestamp
-import java.time._
 import java.util
 
 import org.apache.spark.SparkContext
 import org.apache.spark.sql.Row
 import org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema
-import org.apache.spark.sql.catalyst.util.DateTimeUtils
 import org.apache.spark.sql.types.StructType
-import org.neo4j.driver.v1.{Driver, Session, StatementResult, Transaction, TransactionWork}
 import org.neo4j.spark.dataframe.CypherTypes
+import org.neo4j.spark.utils.Neo4jSessionAwareIterator
 
 import scala.collection.JavaConverters._
 
 
 object Executor {
 
-  def convert(value: AnyRef): Any = value match {
```

Comment:
what happend with this?

Reply:
I'm sorry something went wrong when I rebased the code.
I'll re-add this and others part